datafusion/core/src/prelude.rs

```diff
@@ -39,7 +39,6 @@ pub use datafusion_expr::{
     Expr,
 };
 pub use datafusion_functions::expr_fn::*;
-pub use datafusion_functions_aggregate::expr_fn::*;
```

Comment:
expr / sum is not removed in this PR, so I need to remove this to avoid import conflict